precompiles/assets/assets.go

```diff
@@ -3,10 +3,9 @@ package assets
 import (
 	"bytes"
 	"embed"
-	"fmt"
 	"math/big"
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
+	"golang.org/x/xerrors"
```

Comment:
can you explain why use this third package not use standard library fmt ? do you want stack trace information ?